File: Selenium/Activities/Activity20.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    driver.get("https://v1.training-support.net/selenium/javascript-alerts")

    print(driver.title)

    driver.find_element(By.ID, "prompt").click()

    prompt_alert = driver.switch_to.alert

    print(prompt_alert.text)
    
    prompt_alert.send_keys("Python")

    prompt_alert.accept()

    # confirm_alert.dismiss()

    driver.close()

File: Selenium/Activities/Activity20.java
package activities;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity20 {

	public static void main(String[] args) {

		WebDriver driver =  new FirefoxDriver();
		
		driver.get("https://v1.training-support.net/selenium/javascript-alerts");
		
		System.out.println(driver.getTitle());
		
		driver.findElement(By.id("prompt")).click();

        Alert promtAlert = driver.switchTo().alert();

        String alertText = promtAlert.getText();
        System.out.println("Text in alert: " + alertText);

        promtAlert.sendKeys("Awesome!");

        promtAlert.accept();

        // promtAlert.dismiss();
        
        driver.close();

	}

}
